src/lib/types/validator.ts

```diff
@@ -39,6 +39,11 @@ export const zValidatorData = z
   .transform(({ website, ...val }) => ({
```

Comment:
let's do `consensus_address: zConsensusAddr` which we already have it in GQL

then in `zBlockDataResponseLcd` instead of having `rawProposerAddress`, add `consensusAddress` instead which you can derive as below:
```
const data = fromBase64(blockData.rawProposerAddress);
console.log(toBech32(`terravalcons`, data));
```

for `zValidatorInfoLcd`, we can derive `consensusAddress` by:
```
// for ed25519
const data = fromHex(
    toHex(sha256(fromBase64(found.consensusPubkey.key))).slice(0, 40)
);
console.log("from pubkey", toBech32(`terravalcons`, data));

// for secp256k1 (haven't tested yet tho, but should be correct)
const data = new Ripemd160().update(sha256(pubkey)).digest();
console.log("from pubkey", toBech32(`terravalcons`, data));
```

Reply:
I would rather not use Bech32 prefix for creating a consensus address.

Reply:
hmm, why tho? we can have consistent type for `ValidatorData`